src/factors/flux/FluxModelsPose2Pose2.jl

```diff
@@ -42,14 +41,13 @@ end
 
 function calcVelocityInterPose2!( nfb::FluxModelsPose2Pose2,
                                   iPts::AbstractMatrix{<:Real},
```

Comment:
do these not become `::AbstractVector`?  Matrix was for old pre-CalcFactor i think

Reply:
I updated in MixutreFlux, and moved this one to the attic